ament_cpplint/ament_cpplint/cpplint.py

```diff
@@ -141,6 +142,12 @@
       Examples:
         --linelength=120
 
+    exclude=dir1 dir2 file1...
+      List of exclude directories or files for cpplint
+
+      Examples:
+        --excludedir=LinearMath impl
+
```

Comment:
This comment (https://github.com/ament/ament_lint/pull/234#issuecomment-625991937) still applies.

Reply:
Done https://github.com/ament/ament_lint/pull/238/commits/b9f07a6de02a3eca53bd82b4562b7748dcb14166